webdriver-spec.html

```diff
@@ -2731,6 +2731,16 @@
 
  <li><p>Set the <a>webdriver-active flag</a> to true.</p>
 
+ <li><p>Set the <a>current top-level browsing context</a>
+  for <var>session</var> in an implementation-specific way.
```

Comment:
_Current top-level browsing context_ is a WebDriver concept, and not a browser concept. This state is what WebDriver implementations use to track which browsing context to interact with.

What we need to say is that the _current browsing context_ is the _top-level browsing context of the UA’s currently selected browsing context_, or something along those lines.